src/main/java/frc/robot/RobotContainer.java

```diff
@@ -130,6 +135,9 @@ _gyro, new ModuleIOSparkMax(Constants.CAN.MODULE_FL_DRIVE, Constants.CAN.MODULE_
 
         // Configure the button bindings
         configureButtonBindings();
+        new Dashboard(_shooterBed, _notepath, _shooterFlywheel, _drive, _intake, _climb);
```

Comment:
Let's create a member variable for this. You'll have a warning about this being unused for now. We can currently suppress it with the same `@SuppressWarnings...` annotation we're using for the vision subsystem. Then, let's move the creation of the dashboard right before we make the `_autoChooser` (right after the end of the `switch (Constants.AdvantageKit.CURRENT_MODE)` block.

```java
@SuppressWarnings("unused")
private final Dashboard _dashboard;
```

```java
_dashboard = new Dashboard(_shooterBed, _notepath, _shooterFlywheel, _drive, _intake, _climb);
```